custom-connectors/Staffbase/apiProperties.json

```diff
@@ -0,0 +1,43 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API-Schl\u00fcssel",
```

Comment:
@Nm-jens Please replace the unicodes with English language characters in all the connector artifacts as they would not be supported in all the Power Automate pages.